azure-pipelines.yml

```diff
@@ -35,11 +35,11 @@ steps:
 
   - bash: |
       case $AGENT_OS in
-        "Linux") sudo apt-get install ocaml ;;
-        "Darwin") brew install ocaml ;;
+        "Linux") sudo apt-get install ocaml python3-sphinx;;
```

Comment:
I suggest making it a separate job, say  "Install sphinx". Small one-purpose jobs make it faster and easier to locate and understand CI failures.  Although, the installation is probably of sphinx is probably unlikely to fail..

Reply:
Sure, I can do that. Let's see if the current jobs work, and I'll add it in after.